packages/python-asyncio-throttle/PKGBUILD

```diff
@@ -0,0 +1,28 @@
+# This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
+# See COPYING for license details.
+
+pkgname=python-asyncio-throttle
+_pkgname=asyncio-throttle
+pkgver=1.0.2
+pkgrel=1
+pkgdesc='Simple, easy-to-use throttler for asyncio.'
+arch=('any')
+url='https://pypi.org/project/asyncio-throttle/'
+license=('MIT')
+depends=('python')
+makedepends=('python-setuptools')
+source=("https://files.pythonhosted.org/packages/source/${_pkgname::1}/$_pkgname/$_pkgname-$pkgver.tar.gz")
```

Comment:
Can you use `${_pkgname:0:1}`, I read that `${_pkgname::1}` is not very retro-compatible.